src/style.css

```
@@ -0,0 +1,43 @@
+* {
+  margin: unset;
+}
+
+body div {
+  display: flex;
+}
```

Related to previous comment
```suggestion
.stars {
  display: flex;
}
```